aws-fargate-cfn/op-scim-bridge.yaml

```diff
@@ -0,0 +1,392 @@
+AWSTemplateFormatVersion: 2010-09-09
+Description: >-
+  Deploys 1Password SCIM bridge using Amazon ECS and AWS Fargate. Provisions and
+  configures a VPC, 2 public subnets, an internet gateway, a route table, an
+  ALB, an ACM certificate, Route 53 DNS records, an AWS secret to store
+  credentials for your SCIM bridge, security groups, and required IAM roles.
+Metadata: 
+  AWS::CloudFormation::Interface: 
+    ParameterGroups:
+      - 
+        Label: 
+          default: ""
+        Parameters: 
+          - VPCCIDR
+          - Route53HostedZone
+          - DomainName
+          - SCIMBridgeVersion
+          - scimsession
+    ParameterLabels: 
+      VPCCIDR: 
+        default: "VPC CIDR"
+      Route53HostedZone:
+        default: "Route 53 hosted zone"
+      DomainName:
+        default: "1Password SCIM bridge domain name"
+      SCIMBridgeVersion:
+        default: "1Password SCIM bridge version"
+Parameters:
+  VPCCIDR:
+    Type: String
+    Default: 10.0.0.0/16
+    Description: A CIDR block for the VPC to be created
+  Route53HostedZone:
```

Comment:
Perhaps we should rename the parameter to Route53HostedZoneID to make this clear that the parameter is the Zone ID and not the Zone Name.